sensor/common/networkflow/updatecomputer/transition_based.go

```diff
@@ -280,17 +282,18 @@ func (c *TransitionBased) ComputeUpdatedEndpointsAndProcesses(
 	var procUpdates []*storage.ProcessListeningOnPortFromSensor
 
 	// Process currently enriched entities one by one, categorize the transition, and generate an update if applicable.
+	h := xxhash.New()
 	for ep, p := range enrichedEndpointsProcesses {
 		currTS := p.LastSeen
-		epBinaryKey := ep.BinaryKey()
-		epHashKey := ep.BinaryKey() // Same as epBinaryKey since we only support binary now
+		epBinaryKey := ep.BinaryKey(h)
+		epHashKey := ep.BinaryKey(h) // Same as epBinaryKey since we only support binary now
```

Comment:
This is some uncleaned refactor leftover - there is no need to call it twice as the value will be the same. Let's drop one of them and use the other one.